### PR TITLE
ignore some storybook on Chromatic

### LINKED ISSUE #3474: Why all the ChromaticQA tests in this repo?!!!
**Is your feature request related to a problem? Please describe.**
We pay per snapshot. We have a lot of snapshot tests in ChromaticQA, every single story!

**Describe the solution you'd like**
Why don't we just test the full pages in this repo and the components separately in https://github.com/bbc/psammead/ ?

We can use this https://docs.chromaticqa.com/ignoring-elements

**Describe alternatives you've considered**
Nothing this seem pretty obvious, it also should discourage people creating components in Simorgh

**Testing notes**
Make sure everything in simorgh unchromaticqa is covered in psammead chromaticqa.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
This would save circa 5 mins of Jenkins runtime per push to the repo!!!!

There's also absolutely no need to run these tests on the latest branch, so we should just turn this off right away.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #3474

**Overall change:** _Ignoring components that overlap in the main page storybook stories_

**Code changes:**

- _addParatemers of `{ chromatic: { disable: true } }` to stories so that they do not run on chromatic_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by j-pendlebury — APPROVED** [review]

File: src/app/containers/ErrorMain/index.stories.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { ServiceContextProvider } from '#contexts/ServiceContext';
 import ErrorMain from '.';
 
 storiesOf('Pages|Error Page', module)
+  .addParameters({ chromatic: { disable: true } })
```
Comment (chris-hinds) [inline src/app/containers/ErrorMain/index.stories.jsx]: I think we should continue to test our error pages.

**Review by chris-hinds — CHANGES_REQUESTED** [review]

**Comment by amywalkerdev** [comment]
Khoa is off for two weeks, can someone else pick this up?

**Comment by chris-hinds** [comment]
I'll pick this up. It will need another review once done.

File: src/app/containers/ErrorMain/index.stories.jsx (hunk 1 above)
Comment (amywalkerdev) [reply src/app/containers/ErrorMain/index.stories.jsx]: Removed this from ErrorMain

**Review by ryanmccombe — APPROVED** [review]

**Comment by chris-hinds** [comment]
Thanks @amywalker1987 👍

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**